src/jepsen/voltdb/runner.clj

```diff
@@ -75,6 +75,16 @@
                                  (or (nemeses nem)
                                      (special-nemeses nem))))
                (cli/one-of (concat nemeses (keys special-nemeses)))]]
+    
+    ["-k" "--kfactor NUMBER" "K-factor for database installation"
+    :parse-fn #(Integer/parseInt %)
+    :default 1
```

Comment:
To keep with the previous default, kfactor default should be number of hosts -1

Reply:
one option into the test can not be computed based on the value of other options. At this point I do not know the number of nodes. 
I can assign this default somewhere else. But I am not sure it make sense to have n-1 k-safety. It is not our default installation.